e2e/test/scenarios/dashboard-filters/parameters.cy.spec.js

```diff
@@ -86,9 +86,7 @@ describe("scenarios > dashboard > parameters", () => {
     });
     selectDashboardFilter(getDashboardCard(1), "Category");
 
-    // finish editing filter and save dashboard
-    // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-    cy.contains("Save").click();
+    saveDashboard();
```

Comment:
Can we please get rid of this change, and only focus on the build scripts?

Reply:
sure, I don't know any other way to trigger e2e tests

Reply:
Fair enough.
Let's merge it as is, then.